cli/azd/pkg/auth/cache_windows.go

```diff
@@ -104,5 +147,15 @@ func (c *encryptedCache) Set(key string, val []byte) error {
 		return fmt.Errorf("failed to free encrypted data: %w", err)
 	}
 
-	return c.inner.Set(key, cs)
+	toStore, err := json.Marshal(envelopedData{
+		Type: cCryptProtectDataEncryptionType,
+		Data: base64.StdEncoding.EncodeToString(cs),
+	})
+
+	// We never expect the above to fail.
+	if err != nil {
+		panic(fmt.Sprintf("failed to marshal enveloped data: %s", err))
+	}
+
+	return c.inner.Set(key, toStore)
```

Comment:
Can we support an azd-config to opt-out from encoding token cache?  
It is really useful, as developers, if we want to inspect the auth bytes to be able to manually update the cache, just like we have in Linux
![image](https://user-images.githubusercontent.com/24213737/234715588-459697b0-c2a0-45f1-bf1d-faf5d54847ad.png)

It's not a request for this PR, but a question about what you think about it @ellismg  :)..

Reply:
I'll look at doing a follow up PR that supports something like `AZD_DEBUG_UNSAFE_MSAL_CACHE_STORAGE` or something to let us opt out of this behavior or perhaps write a small tool in a gist that just decrypts and dumps the cache so we can take a look at it.

I have wanted this a few times when making changes in this area, so I can see the value to us as individual developers, but I don't want to make it too easy to fall into the trap of having something unencrypted without really knowing you are going to do that.